plugins/modules/net_tools/nmcli.py

```diff
@@ -582,6 +595,20 @@
 #     - 8 NetworkManager is not running
 #     - 9 nmcli and NetworkManager versions mismatch
 #     - 10 Connection, device, or access point does not exist.
+
+- name: Create the wifi connection
+  community.general.nmcli:
+    type: wifi
+    conn_name: Brittany
+    ifname: wlp4s0
+    ssid: Brittany
+    wifi_sec:
+      key-mgmt: wpa-psk
+      psk: my_password
+    autoconnect: true
+    state: present
+  no_log: true
```

Comment:
Is this still needed?

Reply:
I think we should keep it, it's a very standard use-case. I had to actually refrain myself from changing the rest of the examples to something more standard.

Reply:
I mean the `no_log: true` (in case you meant the whole example).

Reply:
OH! Yes, I can remove the line.